proto/ibc/applications/fee/v1/ack.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package ibc.applications.fee.v1;
+import "gogoproto/gogo.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+// IncentivizedAcknowledgement is the acknowledgement format to be used by applications wrapped in the fee middleware
+// It contains the raw acknowledgement bytes, as well as the forward relayer address
+message IncentivizedAcknowledgement {
```

Comment:
```suggestion
message Acknowledgement {
```
Perhaps maybe just use `Acknowledgement`? This type will always be referenced under its import path "github.com/cosmos/ibc-go/modules/apps/29-fee/types", so it won't collide with the channeltypes naming

Reply:
what would be the benefit of using Acknowledgement here? this is a custom Ack type which we will be using only if the versioning determines that the fee middleware is being used, so i think it would make sense to still keep it as a custom ack type

Reply:
I think he just means naming here, its already a custom type since its in the `fee.proto` file. But I feel like its fine to explicitly call it `IncentivizedAcknowledgement` or `IncentivizedAck` so that its very clear what its purpose is